Formula/dislocker.rb

```diff
@@ -0,0 +1,25 @@
+class Dislocker < Formula
+  desc "FUSE driver to read/write Windows' BitLocker-ed volumes"
+  homepage "https://github.com/Aorimn/dislocker"
+  url "https://github.com/Aorimn/dislocker/archive/v0.7.1.tar.gz"
+  sha256 "742fb5c1b3ff540368ced54c29eae8b488ae5a5fcaca092947e17c2d358a6762"
+
+  bottle do
+    sha256 "a7af4bb7e8615632eb6dfd0e74dbfbb7933f98f1723e588e24bd9e7d2621a770" => :high_sierra
+  end
+
+  depends_on "cmake" => :build
+  depends_on "ruby@1.8" => :build
```

Comment:
It seriously only builds using Ruby 1.8?

Reply:
Pretty sure this wouldn't just be a `:build` dep either?

Reply:
I found a way to disable ruby dependency